docs/en/advanced/Namespace-Management.md

```diff
@@ -286,6 +286,56 @@ $ ls /tmp/alluxio-demo
 hello
 ```
 
+### Metadata Active Sync for HDFS
+In version 2.0, we introduced a new feature for maintaining synchronization between Alluxio space and the UFS when the UFS is HDFS.
+The feature, called active sync, listens for HDFS events and periodically synchronizes the metadata between the UFS and Alluxio namespace as a background task on the master.
+
+To designate a directory for active syncing, issue the following Alluxio command.
```

Comment:
Could we specifically stick to the phrase `active sync` and use it only as a noun? For example `To enable active sync on a directory ...`

Reply:
done